atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,226 @@
+            - Storing the association in the database
+
+        Note:
+            Updates the COMPETENCY collection with new embeddings and cluster associations.
+        """
+
+        clusters = self.weaviate_client.get_all_embeddings(CollectionNames.CLUSTERCENTER.value)
+        medoids = np.array([entry["vector"] for entry in clusters])
+        competencies = self.weaviate_client.get_all_embeddings(CollectionNames.COMPETENCY.value)
+
+        for competency in competencies:
+            uuid, embedding = generate_embeddings(competency["properties"]["competency_id"], competency["properties"]["text"])
```

Ruff (0.11.9)

Line too long (126 > 88)

(E501)